UrbanMobility/AccessPoint/doc/spec.md

```diff
@@ -0,0 +1,50 @@
+# gtfs:AccessPoint
+
+## Description
+
+See [https://developers.google.com/transit/gtfs/reference/#stoptxt](https://developers.google.com/transit/gtfs/reference/#stoptxt)
+
+It is a GTFS `stop` which `location_type` is equal to `2`.
+
+## Data Model
+
+### Entity id
+
+It shall be `urn:ngsi-ld:gtfs:AccessPoint:<access_point_identifier>`
+
+being `access_point_identifier` a value that can derived from the `stop_id` field. 
+
+### Entity Type
+
+It shall be equal to `gtfs:AccessPoint` 
+
+### Properties
+
+Same as `gtfs:Stop`. 
+
+### Relationships
```

Comment:
i would use attribute, as for the old data model template, and the in the attribute you specify that it correspond to a property or a relation in linkeddata.

Reply:
I would prefer something like

# Attributes

## Properties

## Relationships

I will send a PR including a proposal for a new template

Reply:
i am fine, but still i think that the existing format is more informative. so i would describe each "attribute" as we do now, and adding specific bullets for the reference to linked data schemas.
the table is also fine, but i would keep it as a summary. this will allow the model to be well described in the repository, rather than people having to go around like crazy to "rebuild" them.